### PR TITLE
add pagination

### PR DESCRIPTION
**Pull request by MatejaC**

### CODE REVIEW

File: src/components/Results.tsx [hunk 1]
```diff
@@ -1,31 +1,58 @@
-import { Grid } from '@mui/material';
+import { Box, Container, Grid, Pagination } from '@mui/material';
 import ResultCard from './ResultCard';
 import { Result, ResultList } from '../types/result';
+import { useState } from 'react';
 
-type ResultsProps = {
+interface ResultsProps {
```
Comment (e1401) [inline src/components/Results.tsx:6]: How come this was changed from `type` to `interface`?

File: src/components/Results.tsx [hunk 2]
```diff
@@ -1,31 +1,58 @@
...
   searchResults: ResultList;
-};
+}
 
 const Results = ({ searchResults }: ResultsProps) => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [itemsPerPage] = useState(10);
```
Comment (e1401) [inline src/components/Results.tsx:12]: `useState` hook returns a value variable and a setter function (exactly like `currentPage` and `setCurrentPage`) but here you only declared a value without a setter, and does this value of 10 need to be stored as state?

Comment (MatejaC) [reply src/components/Results.tsx:12]: yeah, no need for useState...

File: src/components/Results.tsx (hunk 1 above)
Comment (MatejaC) [reply src/components/Results.tsx:6]: no reason, really;)

File: src/components/Results.tsx [hunk 3]
```diff
@@ -1,31 +1,58 @@
...
+          <Pagination
+            count={Math.ceil(searchResults.length / itemsPerPage)}
+            page={currentPage}
+            onChange={(_, newPage) => setCurrentPage(newPage)}
+            showFirstButton
+            showLastButton
+            color='primary'
+            size='small'
+            sx={{
+              margin: '20px 0px',
```
Comment (e1401) [inline src/components/Results.tsx:50]: margin: '20px 0px' is quite the same as margin y or  `my: '20px'`

**Review by e1401 — APPROVED** [review]
Really good work!

File: src/components/SearchBox.tsx [hunk 4]
```diff
@@ -40,7 +40,7 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
       const {
         data: { results },
       } = await axios.get(
-        API_URL + `?term=${searchValue}&entity=album&limit=10`
+        API_URL + `?term=${searchValue}&entity=album&limit=50`
```
[the remark this answers is not in the document]
Comment (MatejaC) [reply src/components/SearchBox.tsx:43]: Great! :D

**Review by e1401 — APPROVED** [review]
Good to go 🥳

**Pull request merged**